docs/reference_guide.md

```diff
@@ -3401,6 +3401,7 @@ END
 - `print(value)` - Print a value
 - `clear(@x)` - Delete all keys from the map
 - `zero(@x)` - Set all map values to zero
+- `size(@x)` - Get the number of elements in the map
```

Comment:
nit for language consistency with the adoc: "Return the number of elements in the map."

And super nitpicky, I prefer the present tense "returns" so it reads like "what does 'size' do? It 'returns the number of elements in the map'." - but that would involve changing these other ones so I settle for "return".

Reply:
Agreed, let's use "return" as everywhere else.